keylime.conf

```diff
@@ -23,7 +23,7 @@ revocation_notifier_ip = 127.0.0.1
 revocation_notifier_port = 8992
 
 # turn on or off TLS keylime wide
-enable_tls = True
+enable_tls = False
```

Comment:
We can't disable it by default, if its for testing development that you need this off, I recommend you have debug value that sets this accordingly if recognised as running for testing, an example here the @nabilschear and @jetwhiz used for a value when running in their IDE:

https://github.com/keylime/keylime/blob/7c6e0ea539d3952991e77eaefb57fbf14b83ea4e/keylime/provider_platform_init.py#L54-L55

Reply:
OK, I'll follow that. I don't know we can do this for development. I'll look into that.